rocky/rocky/templates/partials/elements/ooi_tree_condensed_table.html

```diff
@@ -6,6 +6,7 @@
             <table class="action-buttons tree-view">
                 <thead>
                     <tr>
+                        <th class="col-ooi-child-nodes">{% translate "Degree" %}</th>
```

Comment:
Degree is not very logical here I'd think. Could we call it 'Child count' instead?